packages/chatbot-server-mongodb-public/src/processors/retrieveRelevantContent.ts

```diff
@@ -33,6 +33,7 @@ export const retrieveRelevantContent = async function ({
 
   const { content, queryEmbedding } = await findContent({
     query: searchQuery,
+    filters: {},
```

Comment:
can you make this an optional property? 

that way you don't need to pass the empty obj

```suggestion
```